ubuntudesign/www.ubuntu.com
# Restructure settings and requirements

Inline with the standards for our other sites, and in preperation for standardising the local development tooling:

- Use a single requirements.txt file
- Use a single settings.py file, which reads environment variables `DJANGO_DEBUG` and `SECRET_KEY`

**NB:** Please review and approve this, but don't merge. Releasing it will require an update to the deployment spec, so we should only merge this when the new spec is ready.

QA
---

``` bash
make clean-all run
docker-compose run sass
```

Check the site still works.
